packages/dcos-signal/extra/dcos-signal.service

```diff
@@ -14,4 +14,4 @@ ExecStartPre=/bin/ping -c1 leader.mesos
 ExecStartPre=/opt/mesosphere/bin/bootstrap dcos-signal
 
 # Signal depends on the dcos-licensing service for the license_id
-ExecStart=/opt/mesosphere/bin/dcos-signal -licensing-socket /run/dcos/dcos-license.sock -segment-key 51ybGTeFEFU1xo6u10XMDrr6kATFyRyh
+ExecStart=/opt/mesosphere/bin/dcos-signal -licensing-socket /run/dcos/dcos-licensing.sock -segment-key 51ybGTeFEFU1xo6u10XMDrr6kATFyRyh
```

Comment:
I'm a bit confused why we would need the licensing-socket at all in dcos/dcos since dcos-licensing isn't even running there but only in EE, is it?

Reply:
Ya, that's a fair point. The reasoning was that since dcos-signal is in open source, this change should be there as well. But I think you're right, we should just move this to enterprise. I'll close this PR and make the change in enterprise instead.